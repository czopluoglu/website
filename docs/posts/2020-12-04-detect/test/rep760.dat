01000000111111101101
10110111000001001110
00011111110001101110
00001000000000011000
11001001000010001110
00011100010110100010
00001001001010010011
11001011111000000110
01101110011100001010
00000000001000100011
11101111111010100000
10000101001001100100
11110000100110110111
00000111001000111101
11000000001000000000
00011001101110100011
00001001010000100111
10000001010000000010
10000101011111111110
11111010101110001011
00101000010111001110
11100011111111100101
11101001010111001110
10111010110001000011
10000000001011110101
01100100000110011111
10010010010100010110
10110001011010110101
01001001001001101110
01110101011110111011
01001001001010111111
00010100100011011111
10101110111010100001
10010000010111000011
00001101001010011011
11111111111111101011
11101101000010101011
11011001110010000111
11111011100011101110
10001000001110110011
10001110111111111111
10010010101011111101
10001100011010100111
11001111011010101110
11101001101100111101
11110001111011001110
10100101111110011110
00110001010111001100
01001000101010010101
10100100000000110010
10101001010110101010
11100011111101101111
11111111111111111111
00011001011011101101
10001010111111000001
01000001101101110111
10010011000011110010
00001000101111000111
00001011011111101011
11000101001110111110
11100000001111101111
01101000111001111010
10110001001100010111
10110101111100110000
01110011011010111001
10001111101110101111
11110001101111000110
01010000101010111010
10010000101010001001
10000010001000000011
11100001010010001010
11000111010110110110
11111111011111101111
01000000001000000001
11110010101010000010
00001101101010011001
10111111110001010011
11010011011001101010
11001100011111001110
01100100011010000010
10101000001110010011
10101111111000101010
11010111011011000111
00010001010101111010
01010001000110110001
01111111101010000101
11011001000100001010
01101011010110111111
11100111101110111110
01111111110100111011
00010100100010011010
00000000001010000100
10111001011000000010
10010010101110000110
11000000011111110000
00000001011101001011
11001101111011101011
00110001011000000100
11000101010111101110
00001001000011100110
00010001011010000011
11111001000000000110
00000000001110100100
01000100011110111110
10000000001010100110
00101101111010111011
11001101011110101011
00110100010011110110
01111001101010101111
11110111101010111111
10000110010010100011
01100111101011111111
10000000110010100011
11110111111111101010
11000110010110111110
11000100111010010000
00001001101111111101
10000101011010100010
11101111011011111101
10000001101111111110
10000111111110110110
10011010000011010111
00010001010010000001
11101010010110001101
10001000101010000110
11011111111001000011
11000000000010110010
10001111111000101011
11100101001100111111
10100000000011000101
11101001011000000111
01010000011101111100
00000011001011001111
01110101101100100010
00111001001111111111
11110111111110010110
10010001001010100000
00101101100111111110
10011001101010100100
00000101001111100111
01111100101011011101
01100101111110110111
00000001010100001001
10101011110000011110
11001001111000101110
11011011111100111110
11111111100011010000
00101101000101111010
01000000001111101111
00011010111111110111
10111100101101011101
11001111101111001110
00110100101101010111
11010100111110110110
00000000101110000110
10111001011000101101
00001101100111110111
10100111011001101000
10111000111111111010
11010000111010100111
11001100011100000100
11010110110111110111
01110001111111100111
01100010011010011110
00000001011001101110
11011001011111001111
11001110011000101110
10001100011001111111
00010001001111111111
01110100101000111100
10111001100101100100
10110001011111111110
01001011010111111111
10001001111000111101
10000101101111011110
11100011101101010101
11000000010001101111
11000111111100101010
11011101100011101100
10000100110111111111
11111111111010111110
10000001011010001010
11110101000011100100
01100001001111011111
00000001101000100100
10000001000111100110
00000000001111110110
10001001001011011100
11111010011000111111
11101101100100101000
00001000001010101111
10101010101111110111
01011101001110101110
10100101111111110111
11110100101000010011
00001101011011111111
11001100111111101011
11000001111010000010
11101101111111111111
10111100010111110101
11111111110110011010
01010000101001101011
01111101011010000101
00001010101000001111
10000101000100011010
11000001011110101000
10100101111101111111
01111100001110111010
10001110011000101101
10011000011001110111
10001101010010111011
00011001111010000101
10000010111010111111
00001111010101110110
11110101111111111111
01100001111111110111
01011000011110010110
11100001011011101101
10001101011111101011
10001111111010010111
01100111111010001111
11101110110010110001
10101111101111011111
11011011111001110111
00000000011001101000
00011100110001100011
00110111111100111101
01110010011100010010
11101000011100010010
00110000011010001110
00000010111100001011
00110011110110011111
11100000000011111001
11111001000111101010
00000000100001101010
11111101111110001110
10110000100000111111
11001000001011100111
10100001111011010111
11110101101101001111
11110000001000101010
01001100111010010110
10000111101101110110
11010001111110111111
10000000011111011101
00000000001000000000
11100000011011001011
11111101001000001110
10100101101111101010
00100101100110101101
