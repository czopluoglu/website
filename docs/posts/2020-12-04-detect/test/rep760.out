-------------------------------------------------------
                  DETECT SUMMARY OUTPUT
-------------------------------------------------------
 
                    Data File Name: C:/Users/cengiz/Desktop/Github/website/_posts/2020-12-04-detect/test/rep760.dat                                         

              Number of Items used:       20

           Number of Items dropped:        0

               Number of Examinees:      250

                 Minimum Number of 
                Examinees per Cell:        2

         Number of Vectors Mutated:        4

      Maximum Number of Dimensions:       12

                Randomization Seed:    99991

   Minimum percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.00

   Average percentage of examinees
         used after deleting cells
     having less than  2 examinees:    98.83
 
-------------------------------------------------------

 NUMBER OF DIMENSIONS THAT MAXIMIZE DETECT:  5
 
  Exploratory DETECT Statistics:

              Maximum DETECT value:   0.7998
                   IDN index value:   0.7316
                           Ratio r:   0.6121

 PARTITION WITH MAXIMUM DETECT VALUE:

    1    2    2    2    1    3    1    3    2    1
    4    5    5    5    3    5    3    5    1    5

 CLUSTER MEMBERSHIPS:

   -----------CLUSTER  1-------------

    1    5    7   10   19

   -----------CLUSTER  2-------------

    2    3    4    9

   -----------CLUSTER  3-------------

    6    8   15   17

   -----------CLUSTER  4-------------

   11

   -----------CLUSTER  5-------------

   12   13   14   16   18   20
 
   ----------------------------------
 
  Covariance Sign Pattern Matrix:
   d+++-++-++----------
   +d+++---+++-+------+
   ++d+++++++---------+
   +++d++---+-+-+------
   -+++d+---+--+-+-----
   +-+++d++----++-+----
   +-+--+d++++-+-+--+--
   --+--++d+-+---------
   +++---++d+---+---+-+
   +++++-+-+d+++-+--+--
   -+----++-+d++-+---+-
   ---+-----++d+-+-++--
   -+--+++--+++d-+-+---
   ---+-+--+----d----++
   ----+-+--++++-d++++-
   -----+--------+d++++
   -----------++-++d-++
   ------+-++-+--++-d++
   ----------+--+++++d+
   -++-----+----+-++++d
 
 No cross validation for this DETECT run
